docs/drivers/azure.md

```diff
@@ -31,15 +31,27 @@ authenticate:
     Microsoft Azure: To sign in, use a web browser to open the page https://aka.ms/devicelogin.
     Enter the code [...] to authenticate.
 
-After authenticating, the driver will remember your credentials up to two weeks.
+- Browser based authentication **credentials expire in 2 weeks**.
+- Once your credentials expire, you will be prompted to log in again.
+- This is practical for daily use; but not recommended for headless applications.
 
-> **KNOWN ISSUE:** There is a known issue with Azure Active Directory causing stored
-> credentials to expire within hours rather than 14 days when the user logs in with
-> personal Microsoft Account (formerly _Live ID_) instead of an Active Directory account.
-> Currently, there is no ETA for resolution, however in the meanwhile you can
-> [create an AAD account][aad-docs] and login with that as a workaround.
+## Authentication (service principal account)
 
-[aad-docs]: https://azure.microsoft.com/documentation/articles/virtual-machines-windows-create-aad-work-id/
+1. Create an Azure Service Principal account ([described here][sp]) to get a
+   `client_id` and `client_secret`
```

Comment:
@ppadala these are stored on machine's `config.json`. You just need to provide it during `create` command.